FlightJournal.Web/Views/Report/Index.cshtml

```diff
@@ -259,12 +259,9 @@
                             </td>
                         }
                         <td class="hidden-print text-right">
-                            @if (Request.IsPilot() && item.IsCurrentClubPilots())
+                            @if (Request.IsPilot() && item.IsCurrentClubPilots() && ((IEnumerable<Guid>)ViewBag.EditableFlights).Contains(item.FlightId))
                             {
-                                if (User.IsEditor() || item.Date.AddDays(3) >= DateTime.Now)
```

Comment:
`|| item.Date.AddDays(3) >= DateTime.Now`
is present in Flight/PartialGrid.cshtml and in Flight/Index.cshtml

And we also have the comment on the page regarding how long a flight can be edited that might require editing.